how-to/customize-workspace/public/manifest.schema.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This was to make it easy to look up in the browser and it means it will be deployed to the samples and github.io urls if someone wanted to use a url schema.

Reply:
Moved into a schemas folder to not pollute the root directory.